src/kubelet/metric/testdata/kubelet_group_expected.go

```diff
@@ -156,10 +157,10 @@ var ExpectedGroupData = definition.RawGroups{
 			"podName":          "kube-state-metrics-57f4659995-6n2qq",
 			"nodeName":         "minikube",
 			"nodeIP":           "192.168.99.100",
-			//"restartCount": int32(7), // No restartCount since there is no restartCount in status field in the pod fetched from kubelet /pods.
-			//"isReady":              false, // No isReady since there is no isReady in status field in the pod fetched from kubelet /pods.
-			//"status":         "Running", // No Status since there is no ContainerStatuses field in the pod fetched from kubelet /pods.
-			//"startedAt":            parseTime("2018-02-27T15:21:37Z"), // No startedAt since there is no startedAt in status field in the pod fetched from kubelet /pods.
+			// "restartCount": int32(7), // No restartCount since there is no restartCount in status field in the pod fetched from kubelet /pods.
+			// "isReady":              false, // No isReady since there is no isReady in status field in the pod fetched from kubelet /pods.
+			// "status":         "Running", // No Status since there is no ContainerStatuses field in the pod fetched from kubelet /pods.
+			// "startedAt":            parseTime("2018-02-27T15:21:37Z"), // No startedAt since there is no startedAt in status field in the pod fetched from kubelet /pods.
```

Comment:
Maybe we can align the values and comments like the formatter would do if we touch the formatting here already?

Reply:
Done!